e2e/playwright/create-actions/src/tests/create-library.spec.ts

```diff
@@ -79,29 +84,13 @@ test.describe('Create Libraries ', () => {
     randomLibraryDescription = `libraryDescription-${Utils.random()}`;
     libraryDialog = myLibrariesPage.libraryDialog;
     const loginPage = new LoginPage(page);
-    try {
-      await loginPage.loginUser(
-        { username: username, password: username },
-        {
-          withNavigation: true,
-          waitForLoading: true
-        }
-      );
-      await myLibrariesPage.navigate();
-      await myLibrariesPage.selectCreateLibrary();
-    } catch (error) {
-      console.error(`beforeEach failed : ${error}`);
-    }
+    await Utils.tryLoginUser(loginPage, username, username, 'Main beforeEach failed');
+    await myLibrariesPage.navigate();
+    await myLibrariesPage.selectCreateLibrary();
   });
 
   test.afterAll(async () => {
-    try {
-      await sitesApi.deleteSites(createdLibrariesIds);
```

Comment:
`createdLibrariesIds[]` is not used anymore? can be removed?

Reply:
My bad, fixed it. Now this array is added to the Utils method.